components/brave_wallet/browser/BUILD.gn

```diff
@@ -122,6 +122,8 @@ static_library("browser") {
     "//brave/components/brave_wallet/common",
     "//brave/components/brave_wallet/common:common_constants",
     "//brave/components/brave_wallet/common:mojom",
+    "//brave/components/json/rs:cxx",
+    "//brave/components/json/rs:cxx_gen",
```

Comment:
I missed this on review, you should not be depending on the _gen target. The cxx target exports it as a public dep